recipes/scattertext/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "scattertext" %}
+{% set version = "0.0.2.9.8" %}
+{% set sha256 = "5fd562852a403f9f505aa4f4cbce83bf56714ad4f0cd05412232340cfc10698c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win32]
+  entry_points:
+    - scattertext = scattertext.CLI:main
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.0
```

Comment:
@JasonKessler -- Great, I'll update the recipe. Are you interested in being a co-maintainer of this recipe? It just means that you'll have write permission to the feedstock for `scattertext` and can push updates there (although I'm happy to keep the conda-forge recipe up-to-date).